pkg/segment/tracing/handler/jaegerHandler.go

```diff
@@ -0,0 +1,116 @@
+package handler
+
+import (
+	"encoding/json"
+
+	"github.com/siglens/siglens/pkg/ast/pipesearch"
+	segstructs "github.com/siglens/siglens/pkg/segment/structs"
+	"github.com/siglens/siglens/pkg/segment/tracing/structs"
+	"github.com/siglens/siglens/pkg/utils"
+	"github.com/valyala/fasthttp"
+)
+
+type ResponseBody struct {
+	Data   []string `json:"data"`
+	Total  int      `json:"total"`
+	Limit  int      `json:"limit"`
+	Offset int      `json:"offset"`
+	Errors []string `json:"errors"`
+}
+
+type SearchResponse struct {
+	Hits struct {
+		Records []map[string]interface{} `json:"records"`
+	} `json:"hits"`
+}
+
+func ProcessGetServiceName(ctx *fasthttp.RequestCtx, myid int64) {
+
+	requestBody := structs.SearchRequestBody{
+		SearchText:    "SELECT DISTINCT `service` FROM `traces`",
+		StartEpoch:    "now-1h",
+		EndEpoch:      "now",
```

Comment:
lets add a 
`todo: the start and end epochs should be taken from the inbound request`
and also change the startepoch to `now-24h`